stratum/hal/lib/barefoot/bf_chassis_manager.cc

```diff
@@ -57,84 +57,95 @@ BFChassisManager::BFChassisManager(PhalInterface* phal_interface,
       node_id_to_port_id_to_port_state_(),
       node_id_to_port_id_to_port_config_(),
       node_id_to_port_id_to_singleton_port_key_(),
+      node_id_to_port_id_to_sdk_port_id_(),
+      node_id_to_sdk_port_id_to_port_id_(),
       xcvr_port_key_to_xcvr_state_() {}
 
 BFChassisManager::~BFChassisManager() = default;
 
 ::util::Status BFChassisManager::AddPortHelper(
-    uint64 node_id, int unit, uint32 port_id,
+    uint64 node_id, int unit, uint32 sdk_port_id,
```

Comment:
I keep wondering if this would not be better solved by a `BfPort`, like the `BcmPort`: https://github.com/stratum/stratum/blob/9f5bd2b285badbef11e81eca6c31d4a3c4342843/stratum/hal/lib/bcm/bcm.proto#L91

Encapsulates both the ID, node, and sdk port in one message to pass around.

Reply:
Let's address this in the wrapper